exporter/lambda/cross_region_import_replication.py

```diff
@@ -5,92 +5,73 @@
 
 import boto3
 import botocore
-from raven import Client
-from raven.transport import HTTPTransport
+import sentry_sdk
+from sentry_sdk.integrations.aws_lambda import AwsLambdaIntegration
+
+sentry_sdk.init(integrations=[AwsLambdaIntegration()])
```

Comment:
I believe this is what we use in the newer projects: 
```suggestion
sentry_sdk.init(integrations=[AwsLambdaIntegration(timeout_warning=True)])
```